bin/remote-test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -eu
+
+if [ -n "${AXIS_DEVICE_IP}" ]; then
+    CARGO_TEST_CAMERA=${AXIS_DEVICE_USER:-root}@${AXIS_DEVICE_IP}
+    f=`basename $1`
```

Comment:
I like separating input parsing, but I think getting this script merged at all is more important, so I didn't change it.

Reply:
I tested it without `CARGO_TEST_CAMERA` and it works fine. I'm happy either way if you want to change it.